src/main/java/frc/robot/RobotContainer.java

```diff
@@ -101,7 +107,36 @@ private void configureButtonBindings() {
             swerveSubsystem.setSwerveDrivePowers(xPower, yPower, angularPower);
         }, swerveSubsystem));
 
+        moverSubsystem.setDefaultCommand(new RunCommand(() -> {
+            double xPower = -mechController.getLeftX();
+            double yPower = -mechController.getRightY();
+        }, moverSubsystem));
+
         driveAButton.onTrue(new InstantCommand(swerveSubsystem::resetFieldAngle, swerveSubsystem));
+
+        mechYButton.onTrue(new InstantCommand(() ->{
+            if(moverSubsystem.getState() == MoverPosition.GROUND){
+                moverSubsystem.setState(MoverPosition.SUBSTATION);
+            } else {
+                moverSubsystem.setState(MoverPosition.GROUND);
+            }
+        }, moverSubsystem));
+
+        mechRBumper.onTrue(new InstantCommand(() ->{
+            if(moverSubsystem.getState() == MoverPosition.CUBEHIGH){
+                moverSubsystem.setState(MoverPosition.CUBEMID);
+            } else {
+                moverSubsystem.setState(MoverPosition.CUBEHIGH);
+            }
+        }, moverSubsystem));
+
+        mechLBumper.onTrue(new InstantCommand(() ->{
```

Comment:
will onTrue cause this logic to be run thousands of times each time the button is held down? i recall we used to use whenPressed or something

Reply:
The old function used in 2022 was marked as deprecated, onTrue just catched when it switches from false to true. I think whileTrue continuously calls it.